src_dualdesc/POLY_RecursiveDualDesc_MPI.h

```diff
@@ -213,7 +308,7 @@ vectface MPI_Kernel_DUALDESC_AdjacencyDecomposition(
   //
   // Reading the input
   //
-  size_t MaxBuffered = 10000 * n_proc;
+  size_t MaxBuffered = 10000 * size_t(std::sqrt(n_proc));
```

Comment:
What is the theory behind this amazing sqrt(n_proc)?
A little explanation would help.